packages/common/src/discussions/_internal/getPropertyMap.ts

```diff
@@ -12,10 +12,17 @@ export function getPropertyMap(): IPropertyMap[] {
   const map = getBasePropertyMap();
 
   // Type specific mappings
-  map.push({ entityKey: "prompt", storeKey: "data.prompt" });
+  map.push({
+    entityKey: "prompt",
+    storeKey: "data.prompt",
+  });
   map.push({
     entityKey: "location",
     storeKey: "item.properties.location",
   });
+  map.push({
```

Comment:
The `entityKey` and `storeKey` values need flip-flopped.

```jsx
  map.push({
    entityKey: "discussionSettings",
    storeKey: "entitySettings.settings.discussions",
  });
```

Additionally, we need to add the mapping for the `entitySettingsId` so we can discern between create vs update settings requests:

```jsx
  map.push({
    entityKey: "entitySettingsId",
    storeKey: "entitySettings.id",
  });
```